server.lua

```diff
@@ -459,7 +459,24 @@ function SetNickname(user, nickname)
 	if discordId then
 		local name = nickname or ""
 		local endpoint = ("guilds/%s/members/%s"):format(Config.Guild_ID, discordId)
-		local member = DiscordRequest("PATCH", endpoint, json.encode({nick = tostring(name)}))
+		local member = DiscordRequest("PATCH", endpoint, json.encode({nick = tostring(name)}), "Reason for the API call")

```

Comment:
Again, `reason` should be configurable and maybe passed in via the method `SetNickname` ?